Configuration/Settings.yaml

```diff
@@ -4,6 +4,7 @@ Neos:
       Array: 'TYPO3\Eel\Helper\ArrayHelper'
 
     metaDataMapping:
-      title: '${asset.Title || iptc.Title || exif.imageDescription}'
+      title: '${asset.Title || iptc.Title || exif.ImageDescription}'
       caption: '${asset.Caption || iptc.Description}'
-      tags: '${Array.concat(asset.tags, iptc.keywords)}'
\ No newline at end of file
+#      tags: '${asset.Tags}'
```

Comment:
I wanted to show what is happening by default. Nothing but runtime would change by uncommenting that line.